ledger/src/blockstore.rs

```diff
@@ -195,11 +195,23 @@ pub struct IndexMetaWorkingSetEntry {
     did_insert_occur: bool,
 }
 
+/// The in-memory data structure for updating entries in the column family
+/// [`cf::SlotMeta`].
+///
+/// The structure contains two versions of the `SlotMeta`.  `old_slot_meta`
+/// records the latest version of the `SlotMeta` that was persisted in the
+/// blockstore, while `new_slot_meta` is the latest dirty version of the
+/// `SlotMeta` which might not be persisted into the blockstore yet.
 pub struct SlotMetaWorkingSetEntry {
+    // The dirty version of the `SlotMeta` which might not be persisted
+    // to the blockstore yet.
     new_slot_meta: Rc<RefCell<SlotMeta>>,
+    // The latest version of the `SlotMeta` that was persisted in the
+    // blockstore.  If None, it means the current slot is new to the
+    // blockstore.
```

Comment:
Nice additions, only critique is that the explanations look very similar above the struct declaration and above each field declaration. Thinking having one or the other would be sufficient; what do you think?